server.js

```diff
@@ -3,16 +3,24 @@ const app = express();
 const router = express.Router();
```

Comment:
I noticed we're not using router in the main server file. Was it left there in purpose or perhaps we forgot to implement something?